taxonomies/categories.txt

```diff
@@ -32331,14 +32327,30 @@ nl:Drinkyoghurts
 
 <en:Drinkable yogurts
 fr:Yaourts à boire goût fraise, Yaourts à boire fraise, Yaourts à boire saveur fraise
+en:Dairy drink with sugar and fruits, Fermented milk with sugar and fruits, Yogurt with sugar and fruits
```

Comment:
All categories that have <en:Drinkable yogurts as a parent need to be drinkable yogurts.
So here the first name should be
en:Sweetened drinkable yogurts with fruits
fr:Yaourts à boire sucrés aux fruits
And they should not include "Yogurt with sugar and fruits".

Could you take a look at all entries under "Drinkable yogurts"?

Reply:
Done, thanks for the heads up.
Some CIQUAL translations are a bit ambiguous. For instance:
ciqual_food_code:en:19508
ciqual_food_name:en:Dairy drink or fermented milk or yogurt, flavoured, with sugar
ciqual_food_name:fr:Boisson lactée, lait fermenté ou yaourt à boire, aromatisé, sucré